rustls/Cargo.toml

```diff
@@ -22,7 +22,7 @@ ring = { version = "0.17", optional = true }
 subtle = { version = "2.5.0", default-features = false }
 webpki = { package = "rustls-webpki", version = "0.102.1", features = ["std"], default-features = false }
 pki-types = { package = "rustls-pki-types", version = "1", features = ["std"] }
-zeroize = "1.6.0"
+zeroize = "1.7"
```

Comment:
Also not clear why this is needed?

Reply:
Unclear to me too. https://github.com/rustls/rustls/actions/runs/7694066891/job/20964074480 seems to be complaining that our ^1.6.0 is incompatible with aws-lc-rs's ^1.7.0, but:

a) it isn't
b) we asked it to only check direct dependencies only, so aws-lc-rs's dependencies should be irreverent

Reply:
File an issue against `cargo-minimal-versions`?

Reply:
https://github.com/rust-lang/cargo/issues/5657#issuecomment-1914446399